src/test/java/org/fcrepo/camel/SparqlDeleteProcessorTest.java

```diff
@@ -97,6 +98,7 @@ public void testDelete() throws IOException, InterruptedException {
 
         // Test
         template.sendBodyAndHeader(incomingDoc, FCREPO_URI, uri);
+        //@FIXME Not sure why this is blowing up with the camel upgrade.
```

Comment:
should this be addressed prior to merging? 

I played around with it a little to see what is going on, and it appears that `resultEndpoint.expectedBodiesReceived` is expecting to be told what body should be received for every message. Right now there are two messages being sent, but only one value supplied to expectedBodiesReceived, so when the second message is received it hits an IndexOutOfBounds error while trying to compare against a second body.

Anyways, to fix it, it looks like line 92 can be changed to
```
        String expectedBody = "update=" +
                encode("DELETE WHERE { <" + uri + "> ?p ?o }", "UTF-8");
        resultEndpoint.expectedBodiesReceived(expectedBody, expectedBody);
```